client/blocks/login/social.jsx

```diff
@@ -87,6 +104,8 @@ class SocialLoginForm extends Component {
 
 	trackGoogleLogin = () => {
 		this.recordEvent( 'calypso_login_social_button_click' );
+
+		window.sessionStorage.setItem( 'login_redirect_to', this.props.redirectTo );
```

Comment:
Shouldn't we use `localforage` or `store` for storing this value?

Reply:
I explicitly used `sessionStorage` so this value would not persist the user session.
`localforage` is a nice wrapper around different web stores, and has some use cases (need to store a lot of data, compatibility with some browsers etc). But `sessionStorage` is certainly available in all the browsers we support now.
`store` is just a wrapper around `localStorage` so not what we need here.

This comment just made me think about something though. Should we update the comment/like popup flow to use calypso (ie D7368) given that calypso does not support older browsers. The current popup flow probably supports many older browsers.

Reply:
Thanks for the clarification!

> Should we update the comment/like popup flow to use calypso (ie D7368) given that calypso does not support older browsers. The current popup flow probably supports many older browsers.

PCYsg-2bz-p2 mentions our policy regarding browser support for themes, which basically boils down to this: themes should maintain the browser support they launched with. However my understanding is that we want to use the Calypso `Login` page everywhere, and retire the legacy one whenever possible. I don't really like the idea to have to maintain both pages anyway.